android/app/src/fdroid/res/layout/fragment_map.xml

```diff
@@ -9,4 +9,11 @@
         tilesource="Cyclemap"
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
+
+    <ZoomControls
+        android:id="@+id/zoomControls"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center|bottom"
+        android:paddingBottom="50dp"/>
```

Comment:
@the-dagger 
Sure I am on it.
One question though, we need only bottom padding, why should we add padding on other sides?